modules/nf-core/vizgenpostprocessing/preparesegmentation/main.nf

```diff
@@ -0,0 +1,60 @@
+process VIZGENPOSTPROCESSING_PREPARESEGMENTATION {
+    tag "$meta.id"
+    label 'process_single'
+
+    container 'ghcr.io/wehi-soda-hub/vizgen-postprocessing_container:v0.1.1'
+
+    input:
+    val(meta)
```

Comment:
The channel input structure seems a bit off.
The meta map should be linked to the input file you want to describe
Something like `tuple val(meta), path(input_images)` would be more appropriate.
Also would in case parallelization would happen, does the other file would need to be synchronize with the input_images files ? If so you should add them to the tuple.